### PR TITLE
Update app-isolation-overview.md

### PR DESCRIPTION
**Pull request by cchavez-msft**
added a pointer to the repo for bug or feature request filing,

### CODE REVIEW

File: desktop-src/SecAuthZ/app-isolation-overview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ms.date: 10/01/2024
 
 # Win32 app isolation overview
 
-The Win32 app isolation security feature for Windows provides a sandbox environment that can integrated into Win32 apps, providing an additional layer of security. This enhancement requires little to no change to your code.
+The Win32 app isolation security feature for Windows provides a sandbox environment that can integrated into Win32 apps, providing an additional layer of security. This enhancement requires little to no change to your code. If you have a feature request or wish to report a buf, please, file an issue in our GitHub [repo](https://github.com/microsoft/win32-app-isolation/issues).
```
Comment (tiangao-ms) [inline desktop-src/SecAuthZ/app-isolation-overview.md]: I believe it's a `bug`, not a `buf`. Also I'd like to break this line. In `md` file, a new line does not really create a new line

```md
Some sentence.
Another sentence.
```

Renders to 

```
Some sentence. Another sentence.
```

Only
```md
Some paragraph.

Another paragraph.
```

Will render as a new paragraph.

Comment (tiangao-ms) [reply desktop-src/SecAuthZ/app-isolation-overview.md]: Also not sure if the `please` in the middle matches the overall tone of a documentation. I think we should be more objective and less personal.

Comment (cchavez-msft) [reply desktop-src/SecAuthZ/app-isolation-overview.md]: @tiangao-ms I followed your instructions, good sir. Are we good?

**Review by alvinashcraft — APPROVED** [review]
This looks good to me. I can merge when the team is happy with the changes.

**Review by tiangao-ms — APPROVED** [review]

**Pull request merged**